app/src/main/res/layout/deal_list_item.xml

```diff
@@ -1,49 +1,104 @@
 
-        <ImageView
-            android:id="@+id/deal_list_item_image_view"
-            android:layout_width="@dimen/deal_list_item_image_size"
-            android:layout_height="@dimen/deal_list_item_image_size"
-            android:contentDescription="@string/product_image_content_description"/>
+    <TextView
+        android:id="@+id/deal_list_item_price"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:contentDescription="price"
+        android:text="price"
+        android:textSize="18sp"
+        app:layout_constraintLeft_toRightOf="@id/deal_list_item_image_view"
+        app:layout_constraintTop_toTopOf="@id/view"
+        app:layout_constraintBottom_toBottomOf="parent"
+        android:layout_marginHorizontal="20dp"
+        android:paddingVertical="10dp"
+        android:gravity="left"/>
 
-    </LinearLayout>
+    <TextView
+        android:id="@+id/ship"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:contentDescription="price"
+        android:text="ship"
+        android:textSize="18sp"
+        app:layout_constraintRight_toLeftOf="@id/or"
+        app:layout_constraintTop_toTopOf="@id/view"
+        app:layout_constraintBottom_toBottomOf="parent"
+        android:layout_marginHorizontal="3dp"
+        android:paddingVertical="10dp"
+        android:gravity="left"/>
 
-    <LinearLayout
-        android:layout_width="match_parent"
+    <TextView
+        android:id="@+id/or"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_margin="@dimen/activity_horizontal_margin"
-        android:orientation="vertical">
-
-        <TextView
-            android:id="@+id/deal_list_item_title"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:textSize="18sp"
-            android:contentDescription="title" />
-        <LinearLayout
-            android:layout_width="match_parent"
-            android:layout_height="1dp"
-            android:background="@drawable/gray_color"></LinearLayout>
-        <TextView
-            android:id="@+id/deal_list_item_price"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:textSize="18sp"
-            android:contentDescription="price" />
-    </LinearLayout>
-
-</LinearLayout>
+        android:alpha=".5"
+        android:contentDescription="price"
+        android:text="or"
+        android:textSize="18sp"
+        app:layout_constraintRight_toLeftOf="@id/location"
+        app:layout_constraintTop_toTopOf="@id/view"
+        app:layout_constraintBottom_toBottomOf="parent"
+        android:layout_marginHorizontal="3dp"
+        android:paddingVertical="10dp"
+        android:gravity="left"/>
+
+    <TextView
+        android:id="@+id/location"
```

Comment:
naming UI elements can be better. They can be either camel cased with type like `locationTV` or underscore separated like `tv_location`